PicaSWB.js

```diff
@@ -122,7 +122,7 @@ function doExport() {
 		//item.date --> 1100 
 		var date = Zotero.Utilities.strToDate(item.date);
 		if (date.year !== undefined) {
-			writeLine("1100", date.year.toString());
+			writeLine("1100", date.year.toString() + "$n[1977] \n"); // mit jedem Jahrgang Wert in Spitzklammern "$n []" anpassen.);
```

Comment:
standardmäßig soll das Erscheinungsjahr mit dem Berichtsjahr - steht in eckigen Klammern - übereinstimmen, daher habe ich wie folgt geändert
`writeLine("1100", date.year.toString() + "$n[" + date.year.toString() + "]" ); `